pkg/chains/storage/docdb/docdb.go

```diff
@@ -260,7 +263,18 @@ func populateMongoServerURL(ctx context.Context, cfg config.Config) error {
 	// If that doesn't exist, then we move on to `storage.docdb.mongo-server-url`.
 	// If that doesn't exist, then we check if `MONGO_SERVER_URL` env var is set.
 	logger := logging.FromContext(ctx)
-	mongoEnv := "MONGO_SERVER_URL"
+
+	if cfg.Storage.DocDB.MongoServerURLPath != "" {
+		logger.Infof("setting %s from storage.docdb.mongo-server-url-path: %s", mongoEnv, cfg.Storage.DocDB.MongoServerURLPath)
+		mongoServerURL, err := getMongoServerURLFromPath(cfg.Storage.DocDB.MongoServerURLPath)
+		if err != nil {
+			return err
+		}
+		if err := os.Setenv(mongoEnv, mongoServerURL); err != nil {
+			return err
+		}
+		return nil
```

Comment:
For consistency, I think we should either `return nil` in the other if-block in this function:

```go
if cfg.Storage.DocDB.MongoServerURLPath != "" {
  ...
  return nil
}

if cfg.Storage.DocDB.MongoServerURLDir != "" {
  ...
  return nil
}

if cfg.Storage.DocDB.MongoServerURL != "" {
  ...
  return nil
}
```

or chain them with if-else-blocks:

```go
if cfg.Storage.DocDB.MongoServerURLPath != "" {
  ...
} else if cfg.Storage.DocDB.MongoServerURLDir != "" {
  ...
} else if cfg.Storage.DocDB.MongoServerURL != "" {
  ...
}
```